docker/CMSRucioClient/scripts/updateDDMQuota

```diff
@@ -10,43 +10,76 @@ client = Client()
 # NOTE: This probably needs to be reviewed after an assement of age of dynamic data at different sites
 # and if this can be used to normalise that
 
+
 DRY_RUN = False
+VERBOSE = True
+MAX_DDM_QUOTA = 100
 
 # Adding ddm_quota attribute to all disk RSEs
 # T3s do not have the "static" usage set, they are quasi-static
 RSE_EXPRESSION = "rse_type=DISK&cms_type=real&tier<3&tier>0"
-
 rses = [rse["rse"] for rse in client.list_rses(rse_expression=RSE_EXPRESSION)]
+ddm_quotas = {}
+overridden_ddm_quotas = {}
 
-for rse in rses:
-    rse_usage = list(client.get_rse_usage(rse))
-
-    static, rucio, unavailable, expired = 0, 0, 0, 0
-    for source in rse_usage:
-        if source["source"] == "static":
-            static = source["used"]
-        if source["source"] == "rucio":
-            rucio = source["used"]
-        if source["source"] == "unavailable":
-            unavailable = source["used"]
-        if source["source"] == "expired":
-            expired = source["used"]
-
-    # Normalise
-    if static == 0:
-        continue # Skip if static is 0
-
-    # ddm_quota is set proportional to percentage of (dynamic + free) space
-    # Apprantly, python integers do not overflow, https://docs.python.org/3/library/exceptions.html#OverflowError
-
-    locked = rucio - unavailable - expired
-    ddm_quota = int((max(static - locked, 0)/static)*1e4)
-
-    # Override ddm_quota for operational purposes
-    rse_attributes = client.list_rse_attributes(rse)
-    if "override_ddm_quota" in rse_attributes:
-        ddm_quota = rse_attributes["override_ddm_quota"]
-    
-    if not DRY_RUN:
+def calculate_ddm_quotas():
+    for rse in rses:
+        rse_usage = list(client.get_rse_usage(rse))
+
+        static, rucio, unavailable, expired = 0, 0, 0, 0
+        for source in rse_usage:
+            if source["source"] == "static":
+                static = source["used"]
+            if source["source"] == "rucio":
+                rucio = source["used"]
+            if source["source"] == "unavailable":
+                unavailable = source["used"]
+            if source["source"] == "expired":
+                expired = source["used"]
+
+        # Normalise
+        if static == 0:
+            continue  # Skip if static is 0
+
+        # ddm_quota is set proportional to percentage of (dynamic + free) space
+        # Apprantly, python integers do not overflow, https://docs.python.org/3/library/exceptions.html#OverflowError
+
+        locked = rucio - unavailable - expired
+        ddm_quota = int((max(static - locked, 0) / static) * 1e4) ** 2
```

Comment:
Given this is being squared. The multiplication with 10k is not required.

Reply:
We should multiple it by a number >= 10, o/w integer casting converts all values to 0, since they are between 0 and 1.